org.jacoco.doc/docroot/doc/integrations.html

```diff
@@ -40,7 +40,7 @@
     </tr>
     <tr>
       <td>Command Line</td>
-      <td><a href="agent.html">JaCoCo Manual</a></td>
+      <td><a href="cli.html">JaCoCo Manual</a></td>
```

Comment:
@Godin Good idea, added the link back (and also since version as for EclEmma) in ebcd1771581c6841003d872dcba933f0fb6fa238 ✅